Tella/Scenes/Settings/Views/Servers/GDrive/CreateDriveFolder.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CreateDriveFolder.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/21/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct CreateDriveFolder: View {
+    @State var fieldContent : String = ""
+    var body: some View {
+        ContainerView {
+            VStack(spacing: 20) {
+                Spacer()
+                GDriveHeaderView(
+                    title: "Create new folder",
+                    subtitle: "Your reports will be uploaded to a new folder on your Google Drive. Choose a name for this folder here."
+                )
+                TextfieldView(fieldContent: $fieldContent, isValid: .constant(true), shouldShowError: .constant(false), fieldType: .text, placeholder: "Folder name")
+                    .padding(.vertical, 12)
+                Spacer()
+                bottomView
+            }.padding(EdgeInsets(top: 0, leading: 15, bottom: 0, trailing: 15))
+            .navigationBarHidden(true)
+        }
+    }
+    
+    var bottomView: some View {
+        BottomLockView<AnyView>(isValid: .constant(true),
```

Comment:
maybe we need to add a condition  (`TextfieldView` is valid) to validate the next button ?